src/components/dashboard/Transactions.vue

```diff
@@ -0,0 +1,173 @@
+<script setup lang="ts">
+import { useConfirm } from "primevue/useconfirm";
+import { useToast } from "primevue/usetoast";
+import {onMounted, ref} from "vue";
+import TransactionDialog from "./TransactionDialog.vue";
+import { useTransactionStore } from '@/stores/transaction';
+
+const confirm = useConfirm();
+const toast = useToast();
+const transaction = useTransactionStore()
+const id = ref<number | undefined>();
+
+const confirmDelete = (id : number) => {
```

Comment:
Business standard when assigning types to typescript variable is `id: {type}` not `id : {type}`. Remove the space after variable name.